steel/src/host/mod.rs

```diff
@@ -45,8 +45,11 @@ pub(crate) type HostEvmEnv<D, H> = EvmEnv<TraceDb<D>, H>;
 impl EthEvmEnv<TraceDb<AlloyDb<Http<Client>, Ethereum, RootProvider<Http<Client>>>>> {
     /// Creates a new provable [EvmEnv] for Ethereum from an HTTP RPC endpoint.
     pub async fn from_rpc(url: Url, number: BlockNumberOrTag) -> Result<Self> {
```

Comment:
Should this now redundant function be removed, or at least marked as `#[deprecated]`?

Reply:
fixed in 4affaae15a32066e132832ddd647ce84c9152969